ethicml/implementations/agarwal.py

```diff
@@ -14,6 +14,7 @@
 from joblib import dump, load
 from sklearn.linear_model import LogisticRegression
 
+from ethicml import ClassifierType, FairnessType, KernelType
```

Comment:
ideally this would be a more specific import so that we don't have to load everything